sal_interfaces/DIMM/DIMM_Events.xml

```diff
@@ -50,6 +50,13 @@
       <Units>second</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>expiresAt</EFDB_Name>
```

Comment:
`validUntil` is another option for you given the Description string you used below.